pyproject.toml

```diff
@@ -1,4 +1,12 @@
+[tool.black]
```

Comment:
These changes feel unrelated, no?

Reply:
Well I need these settings so that then when I run pre-commit  (which was added in the previous MR by @Mildophin ) it doesn't complain about all my crapy formatting :) Is it fine to you to add them now ?

Reply:
mb that's something I should have added, thanks AC